fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -780,6 +781,10 @@ showCharsetStatement
     : SHOW (CHAR SET | CHARSET) ((LIKE pattern=string) | (WHERE expression))?
     ;
 
+showWarningStatement
+    : SHOW WARNINGS (limitElement)?
```

Comment:
SHOW ERRORS statement miss

Reply:
OK , I thought that this pr is only for  SHOW WARNING statement, if need , I will catch up immediately

Reply:
> SHOW ERRORS statement miss

done!